frontend/src/modules/LoginRedux.js

```diff
@@ -13,22 +12,23 @@ export const CHANGE_PASSWORD_SUCCESS = "CHANGE_PASSWORD_SUCCESS";
 export const CHANGE_PASSWORD_FAILURE = "CHANGE_PASSWORD_FAILURE";
 
 // Action Creators
-const setLoginState = loggedIn => ({ type: SET_LOGIN_STATE, loggedIn });
-
 let url = process.env.REACT_APP_DEV_URL || BACKENDURL;
 
-function authenticateAction(userData, dispatch, location, push) {
-  return async function() {
-    if (navigator.cookieEnabled) {
-      localStorage.setItem("ecom_token", userData.token);
-    }
+// function authenticateAction(userData, dispatch, location, push) {
```

Comment:
Remove commented code.

Reply:
We shouldn't remove this commented code yet. This is the original code. I simplified the **authentication action** for now, just to create a basic login flow.

However, I added comments about that.
See commit #https://github.com/code-for-canada/project-thundercat/pull/279/commits/4b74f5d2e297ca06bcd01e2d32a5dfa7a3489ddd